shenyu-examples/shenyu-examples-dubbo/shenyu-examples-apache-dubbo-service/src/main/java/org/apache/shenyu/examples/apache/dubbo/service/impl/DubboTestServiceImpl.java

```diff
@@ -29,6 +30,7 @@
  * DubboTestServiceImpl.
  */
 @Service("dubboTestService")
+@DubboService
```

Comment:
check `shenyu-examples/shenyu-examples-dubbo/shenyu-examples-apache-dubbo-service/src/main/resources/spring-dubbo.xml`.

Reply:
sorry, I will consider this pr, and this nacos config maybe error.